### PR TITLE
sql binding fix

### PR DESCRIPTION
**Pull request by bradymiller**

### CODE REVIEW

**Comment by bradymiller** [comment]
@danora24 ,
Here is your most recent code in 1 commit from https://github.com/openemr/openemr/pull/1608
-brady

File: library/ajax/addlistitem.php [hunk 1]
```diff
@@ -48,27 +51,19 @@
 // determine the sequential order of the new item,
 // it should be the maximum number for the specified list plus one
 $seq = 0;
-$row = sqlQuery("SELECT max(seq) as maxseq FROM list_options WHERE list_id = '" . $list_id . "' AND activity = 1");
+$row = sqlQuery("SELECT max(seq) as maxseq FROM list_options WHERE list_id = '" . $list_id . "' AND activity = 1", array($list_id, $option_id));
```
Comment (bradymiller) [inline library/ajax/addlistitem.php]: @danora24 , this needs to be fixed to following:
```
$row = sqlQuery("SELECT max(seq) as maxseq FROM list_options WHERE list_id = ? AND activity = 1", array($list_id);
```

File: library/ajax/addlistitem.php [hunk 2]
```diff
@@ -48,27 +51,19 @@
...
-                "list_id, option_id, title, seq, is_default, option_value " .
-                ") VALUES (" .
-                "'".$list_id."'".
-                ",'".trim($option_id)."'" .
-                ",'".trim($title). "'" .
-                ",'".$seq."'" .
-                ",'".$is_default."'" .
-                ",'".$option_value."'".
-                ")");
+    "list_id, option_id, title, seq, is_default, option_value ) VALUES ( ?, ?, ?, ?, ?, ?", array($list_id, $option_id, $title, $seq, $is_default, $option_value));
```
Comment (bradymiller) [inline library/ajax/addlistitem.php]: @danora24 , this needs to be fixed to following:
```
"list_id, option_id, title, seq, is_default, option_value ) VALUES ( ?, ?, ?, ?, ?, ?)", array($list_id, $option_id, $title, $seq, $is_default, $option_value));
```

File: library/ajax/addlistitem.php (hunk 1 above)
Comment (bradymiller) [reply library/ajax/addlistitem.php]: done

File: library/ajax/addlistitem.php (hunk 2 above)
Comment (bradymiller) [reply library/ajax/addlistitem.php]: done

**Comment by bradymiller** [comment]
This code was committed here:
https://github.com/openemr/openemr/commit/ff575b618177718dd4e2f31377ec8bb6dd1b9b3c

Thank you @d3sandoval and congrats on your first openemr commit!!!!

**Comment by d3sandoval** [comment]
@bradymiller thanks! But I think you meant to tag @danora24 ;)

**Pull request closed without merging**